### PR TITLE
pkp/pkp-lib#836 settings appearance form

### PR DESCRIPTION
**Pull request by NateWr**
This and the companion PRs for OJS/OMP restructure the Settings > Website > Appearance form. They:

- shift control over the order of items into the app, and move the common form elements into templates that can be pulled in where desired
- remove a few unused settings
- update the language to reflect how the settings are actually used (and better acknowledging that other themes may use them differently)

### CODE REVIEW

File: locale/en_US/manager.xml [hunk 1]
```diff
@@ -280,7 +280,7 @@
 	<message key="manager.setup.homepageImageInvalid">Invalid homepage image format or upload failed. Accepted formats are .gif, .jpg, or .png.</message>
 	<message key="manager.setup.homeTitleImageInvalid">Invalid homepage header title image format or upload failed. Accepted formats are .gif, .jpg, or .png.</message>
 	<message key="manager.setup.layout.theme">Theme</message>
-	<message key="manager.setup.layout.themeDescription">Choose a theme from the available set of theme plugins.</message>
+	<message key="manager.setup.layout.themeDescription">Select a theme for your website. Only themes installed from the plugins settings are listed here.</message>
```
Comment (asmecher) [inline locale/en_US/manager.xml]: This new language is confusing to me. Maybe "New themes may be added from the Plugins setup area"?

File: templates/controllers/tab/settings/appearance/form/header.tpl [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{**
+ * controllers/tab/settings/appearance/form/header.tpl.tpl
```
Comment (asmecher) [inline templates/controllers/tab/settings/appearance/form/header.tpl]: *.tpl.tpl

File: templates/controllers/tab/settings/appearance/form/homepageImage.tpl [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ *
+ * Copyright (c) 2014-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @brief Form fields for uploading a frontend homepage image
+ *
+ *}
+{assign var="upload_image_field_id" value=$uploadImageLinkActions.homepageImage->getId()}
+{fbvFormSection for="$upload_image_field_id" label="manager.setup.homepageImage" description="manager.setup.homepageImageDescription"}
```
Comment (asmecher) [inline templates/controllers/tab/settings/appearance/form/homepageImage.tpl]: Unnecessary quotes. Also, we generally use $lowerCamelCase for template variables (though it's not always totally consistent).

**Comment by NateWr** [comment]
Each of your comments should be addressed in these last commits.

File: locale/en_US/manager.xml [hunk 4]
```diff
@@ -234,7 +234,7 @@
 	<message key="manager.setup.sponsors">Sponsoring Organizations</message>
 	<message key="manager.setup.sponsors.note">You may add a note describing your sponsor relationship and policy; this will appear with the sponsor list on your About page.</message>
 	<message key="manager.setup.contributors.note">You may add a note describing the relationship and policy between you and your sources of support; this will appear with the Sources of Support list on your About page.</message>
-	<message key="manager.setup.styleSheetDescription">You may upload an additional Cascading Style Sheet (CSS) file to change the appearance.</message>
+	<message key="manager.setup.styleSheetDescription">Experienced web developers can upload a CSS file to further customize the journal's appearance.</message>
```
Comment (asmecher) [inline locale/en_US/manager.xml]: "Journal" is OJS-specific; consider moving this string into the application.

**Comment by asmecher** [comment]
There are still some `$underscore_separated_variable_names` which should be `$camelCaseified`; may as well hit those while you're moving that OJS-specific key.

File: locale/en_US/manager.xml (hunk 4 above)
Comment (NateWr) [inline locale/en_US/manager.xml]: Lol, I actually commited a change from "press" to "journal", thinking I'd solved the problem. Clearly had my blinders on there. Think I meant "website".

**Comment by NateWr** [comment]
Ok, hopefully that's all the `$underscore_variables` now.

**Pull request merged**